src/app/parser/util/CheerioPlugin.ts

```diff
@@ -33,7 +33,7 @@ export const listToParagraph = (elem : Cheerio): string => {
 	const text = [];
 	const items = Cheerio(elem).children();
 	items.each((i, el) => {
-		text[i] =  Cheerio(el).text().replace(/\s*\.\s*$/, '');
+		text[i] =  Cheerio(el).text().trim().replace(/\s*\.\s*/g, '. ').replace(/\s*\.\s*$/, '');
```

Comment:
description would make it easier to read :)

Reply:
added :)